src/components/ProductLink/styles.module.scss

```diff
@@ -0,0 +1,51 @@
+.productWrapper {
+  width: 200px;
+}
+.productWrapper:hover {
+  opacity: 1;
+}
+.unavailable {
+  opacity: 0.5;
+}
+.priceContainer {
+  display: flex;
+  gap: 0.25rem;
+}
+.salePrice {
+  color: red;
+}
+
+.prevPrice {
+  text-decoration: line-through;
+}
+
+.imgWrapper {
+  background-color: #dddddd;
+  position: relative;
+  width: 100%;
+  aspect-ratio: 3/4;
+}
+
+.productImg,
+.revealImg {
+  width: 100%;
+  position: absolute;
+  aspect-ratio: 3/4;
+}
+
+.revealImg {
+  display: none;
+}
+
+.imgWrapper:hover .revealImg {
+  display: block;
+  z-index: 1;
+}
+
+.detailsContainer {
+  h5 {
```

Comment:
Remove h5 specific styling, use margins instead of padding when it comes to spacing elements away from one another and apply margins in one direction on each element:
```
<image>
<label margin-top: 20px>
<prices margin-top: 20px>
```

This way, we aren't left with the bottom spacing underneath the label when prices isn't rendered